from common.tests.utils.helpers import BaseTestCase


class ReportingYearTest(BaseTestCase):
    def fail(self):
        raise Exception('aaa')
